.Documentation/articles/innoactive-creator/suspending-interactions.md

```diff
@@ -0,0 +1,5 @@
+# Suspending Interactions
+
+You can let your trainees to interact with a [training object](training-scene-object.md) by adding certain properties to it. By default, objects stay interactive throughout the whole training course: a trainee could push a button and flick a switch at any time. The training feels more real, but it can turn chaotic in complex courses and scenes. To prevent that, you can temporary suspend interactions for a property, and restore it only when your trainee needs it.
+
+You can suspend user interactions with a training property by using the `Lock` behavior. To suspend user interactions with all properties from the start, attach the `LockObjectsOnSceneStart` component to any active object in the scene. A step will automatically restore interactions with all properties that are referenced by its behaviors and conditions. You can tweak this in the `Unlocked Objects` tab of the Step Inspector.
```

Comment:
Suggestion from my side:

```suggestion
Suspending user interactions with training objects is handled automatically for you. A step will restore interactions with all objects, or to be more precise their properties, that are referenced by its behaviors and conditions. Therefore, in a step you can only grab a cube if it is also referenced in a `GrabbableCondition` in this step. You can tweak this behavior in the `Unlocked Objects` tab of the Step Inspector where you can add training objects manually which then also will be unlocked within this step.

To prevent interaction with training objects before the trainer started the training you can add the `LockObjectsOnSceneStart` script to any GameObject within the scene, e.g. the `[COURSE_CONTROLLER]`.
```